vslm/global_object_manager.go

```diff
@@ -0,0 +1,659 @@
+/*
+Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package vslm
+
+import (
+	"context"
+	"time"
+
+	"github.com/pkg/errors"
```

Comment:
Let's not add this dependency.  There is an errors package coming in Go 1.13: https://crawshaw.io/blog/xerrors

This import is also causing CI to fail, since the package isn't vendored.

Reply:
Sure - not real happy with Go error handling, so let's add this back in when we can.

Reply:
:+1: for now let's just keep it consistent.